### PR TITLE
Implement logic to allow 3scale monitoring to be enabled

### PR DESCRIPTION
**Pull request by obrienrobert**
## JIRA
[MGDAPI-49](https://issues.redhat.com/browse/MGDAPI-49)

## Description
3scale 2.9 ships with monitoring resources (new alerts and dashboards) that are disabled by default. This PR introduces allows us to enable monitoring (via a flag in the API Manager CR) when required. For now, the flag is set to false. The 3scale operator version has also been updated to allow this change to be made.

## Steps to verify
- We need to ensure that once we do enable the monitoring flag in the API Manager CR, the correct alerts and dashboards are created. Using this branch, set [this line](https://github.com/integr8ly/integreatly-operator/pull/1200/files#diff-c99ff6dd3de866743c817f7b1dc230b4875e42eba21e5768f34fd92b9f873e32R909) to `true` and complete an install of RHMI
- Once the install is complete, ensure that all of the new 3scale alerts/dashboards were created in Prometheus and Grafana. A list of both can be found [here](https://gist.github.com/obrienrobert/b390ad39d0c4f63d930a094a8ef4a594)
- Ensure that none of the new alerts are firing after install

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/integr8ly/integreatly-operator
 go 1.13
 
 require (
-	github.com/3scale/3scale-operator v0.5.0
+	github.com/3scale/3scale-operator v0.2.1-0.20200730110533-c3b57b704d73
```
[the remark this answers is not in the document]
Comment (obrienrobert) [reply go.mod:6]: @briangallagher no. We are actually upgrading it (although I agree that it does look like it's being downgraded). In order to enable monitoring, we need to have 3scale 2.9 as a dependency in the vendor folder. If you check the tags in the 3scale-operator (https://github.com/3scale/3scale-operator/tags), the last semvar like tag is `v0.5.0`. The above output is the result of running `go mod vendor` using the `3scale-2.9.0-GA` tag.

[the remark this answers is not in the document]
Comment (briangallagher) [reply go.mod:6]: ok, thanks for the explanation @obrienrobert

**Comment by MStokluska** [comment]
Verified it on a cluster, all looks good to me! 
/lgtm

**Pull request merged**